lib/reducers/ActiveUser.js

```diff
@@ -2,10 +2,11 @@
 const ActiveUser = (state = {}, action) => {
   switch(action.type) {
     case 'SET_USER':
-      const { id, email, username } = action.user
-      return { id, email, username }
+      return action.user
     case 'DUPLICATE':
-      return { error: action.error }
+      return { error: action.error, type: action.type }
+    case 'NOT_FOUND':
+      return { error: action.error, type: action.type }
```

Comment:
Could be refactored to a single user error action, such as `USER_ERROR` that holds the error type and error message.